scripts/train.py

```
@@ -0,0 +1,151 @@
+
+sys.path.insert(0, str(Path(__file__).parent.parent))
+from src.features.build_features import engineer_features
+
+class CreditScorer:
+    def __init__(self, model_path='models/model.pkl', scaler_path='models/scaler.pkl'):
+        try:
+            self.model = joblib.load(model_path)
+            self.scaler = joblib.load(scaler_path)
+            features_df = pd.read_csv('models/features.csv', header=None)
+            self.features = features_df.iloc[:, 0].tolist()
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)